_opensearch/rest-api/explain.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Explain
+parent: REST API reference
+nav_order: 140
+---
+
+# Explain
+Introduced 1.0
+{: .label .label-purple }
+
+Wondering why a specific document ranks higher (or lower) for a query? You can use the explain API for an explanation of how the relevance score (`_score`) is calculated for every result.
+
+OpenSearch uses a probabilistic ranking framework called [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25) to calculate relevance scores. Okapi BM25 is based on the original [TF/IDF](http://lucene.apache.org/core/4_0_0/core/org/apache/lucene/search/package-summary.html#scoring) framework used by Apache Lucene.
+
+The explain API is an expensive operation in terms of both resources and time. On production clusters, we recommend using it sparingly for the purpose of troubleshooting.
+{: .warning }
+
+
+## Example
+
+To see the explain output for all results, set the `explain` flag to `true` either in the URL or in the body of the request:
+
+```json
+POST kibana_sample_data_ecommerce/_search?explain=true
+{
+  "query": {
+    "match": {
+      "customer_first_name": "Mary"
+    }
+  }
+}
+```
+
+More often, you want the output for a single document. In that case, specify the document ID in the URL:
+
+```json
+POST kibana_sample_data_ecommerce/_explain/EVz1Q3sBgg5eWQP6RSte
+{
+  "query": {
+    "match": {
+      "customer_first_name": "Mary"
+    }
+  }
+}
+```
+
+## Path and HTTP methods
+
+```
+GET <target>/_explain/<id>
+POST <target>/_explain/<id>
+```
+
+## URL parameters
+
+You must specify the index and document ID. All other URL parameters are optional.
+
+Parameter | Type | Description | Required
+:--- | :--- | :--- | :---
+`<index>` | String | Name of the index. You can only specify a single index. | Yes
+`<_id>` | String | A unique identifier to attach to the document. | Yes
+`analyzer` | String | The analyzer to use in the query string. | No
+`analyze_wildcard` | Boolean | Specifies whether to analyze wildcard and prefix queries. Default is false. | No
+`default_operator` | String | Indicates whether the default operator for a string query should be AND or OR. Default is OR. | No
+`df` | String | The default field in case a field prefix is not provided in the query string. | No
+`lenient` | Boolean | Specifies whether OpenSearch should ignore format-based query failures (for example, querying a text field for an integer). Default is false. | No
+`preference` | String | Specifies a preference of which shard to retrieve results from. Available options are `_local`, which tells the operation to retrieve results from a locally allocated shard replica, and a custom string value assigned to a specific shard replica. By default, OpenSearch executes get document operations on random shards. | No
```

Comment:
I think we want "By default, OpenSearch executes the explain operation on random shards." here